packages/cli/test/functional/test_site/_markbind/layouts/default/head.md

```diff
@@ -2,3 +2,5 @@
 <meta name="default-head-top">
 </head-top>
 <meta name="default-head-bottom">
+
+**Relative Link Test** This is a relative Intra-Site link (see [Tags](../../../testTags.html))
```

Comment:
you want `header.md`, the similar sounding `head.md` is for inserting things into the `<head>` tag =P